fdbserver/DeltaTree.h

```diff
@@ -25,13 +25,27 @@
 #include "fdbclient/FDBTypes.h"
 #include "fdbserver/Knobs.h"
 #include <string.h>
+#include <emmintrin.h>
 
 typedef uint64_t Word;
 // Get the number of prefix bytes that are the same between a and b, up to their common length of cl
 static inline int commonPrefixLength(uint8_t const* ap, uint8_t const* bp, int cl) {
 	int i = 0;
+
```

Comment:
I think this works:
```
#ifdef __SSE4_2__
```

This definitely works, but checks for AVX, which is more than you need:

```
#ifdef __AVX__
```